src/views/signup/SignupView.vue

```diff
@@ -0,0 +1,269 @@
+              />
+              <button
+                class="flex-1 w-full h-11 text-white bg-limegreen-500 rounded-lg"
+                @click="checkName"
+                type="button"
+              >
+                중복 확인
+              </button>
+            </div>
+          </div>
+          <p
+            class="h-3 text-xs"
+            :class="nameErrorMessage ? 'text-red-500' : 'text-transparent'"
+          >
+            {{ nameErrorMessage }}
+          </p>
+        </div>
+
+        <div>
+          <label for="email" class="mb-1 block font-bold">이메일</label>
+          <div class="flex flex-col gap-2">
+            <div class="flex gap-3">
+              <input
+                v-model="email.email"
+                id="email"
+                type="email"
+                placeholder="이메일을 입력해주세요"
+                class="border-2 border-limegreen-500 flex-2 w-full h-11 rounded-lg bg-white px-3 py-3"
+              />
+              <button
+                class="flex-1 w-full h-11 text-white font-thin bg-limegreen-500 rounded-lg"
+                @click="send"
+                type="button"
+              >
+                인증 요청
+              </button>
+            </div>
+            <div class="flex gap-3">
+              <input
+                v-model="verifyEmail.code"
+                id="email-code"
+                type="text"
+                placeholder="인증번호를 입력해주세요."
+                class="border-2 border-limegreen-500 flex-2 w-full h-11 rounded-lg bg-white px-3 py-3"
+              />
+              <button
+                class="flex-1 w-full h-11 text-white font-thin bg-limegreen-500 rounded-lg"
+                @click="verify"
+                type="button"
+              >
+                확인
+              </button>
+            </div>
+            <p
+              class="h-3 text-xs"
+              :class="emailErrorMessage ? 'text-red-500' : 'text-transparent'"
+            >
+              {{ emailErrorMessage }}
+            </p>
+          </div>
+        </div>
+        <div class="flex flex-col gap-2">
+          <div>
+            <label for="password" class="mb-1 block font-bold">비밀번호</label>
+            <input
+              v-model="member.password"
+              id="password"
+              type="password"
+              placeholder="비밀번호 입력"
+              style="font-family: Arial, sans-serif"
+              class="border-2 border-limegreen-500 flex-2 w-full h-11 rounded-lg bg-white px-3 py-3"
+            />
+          </div>
+          <div>
+            <label for="password2" class="mb-1 block font-bold"
+              >비밀번호 확인</label
+            >
+            <input
+              v-model="password2"
+              id="password2"
+              type="password"
+              placeholder="비밀번호 확인"
+              style="font-family: Arial, sans-serif"
+              class="border-2 border-limegreen-500 flex-2 w-full h-11 rounded-lg bg-white px-3 py-3"
```

Comment:
tailwindcss가 우선순위가 밀려서 적용이 안되는 경우가 있는데 그럴 때는 아래 처럼 뒤에 !를 추가해주시면 적용이 됩니다!!
수정요청은 아니고 앞으로 작업하실때 알고 계시면 편할거 같아요!

```suggestion
              class="border-2 border-limegreen-500 flex-2 w-full h-11 rounded-lg bg-white px-3 py-3 font-sans!"
```